src/components/Account.vue

```diff
@@ -184,6 +193,12 @@
         text-align: center;
     }
 
+    .label.address-font {
```

Comment:
I suggest to simply use `.mono` as the class name.

Reply:
I chose `address-font` as it does more than just setting a mono font.

Reply:
Yah, but that's, like, our definition of mono.